### PR TITLE
Add webpack config and labs flag docs for voice messages

### PR DESCRIPTION
**Pull request by turt2live**
**See https://github.com/matrix-org/matrix-react-sdk/pull/5769**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -252,6 +252,16 @@ module.exports = (env, argv) => {
                         outputPath: '.',
                     },
                 },
+                {
+                    test: /encoderWorker\.min\.js$/,
+                    loader: "file-loader",
+                    type: "javascript/auto", // https://github.com/webpack/webpack/issues/6725
+                    options: {
+                        // We deliberately override the name so it makes sense in debugging
+                        name: 'opus-encoderWorker.min.[hash:7].[ext]',
```
Comment (jryans) [inline webpack.config.js:261]: Huh? I don't think I am following... Why not just name it `opus-...` from the start...?

Comment (jryans) [reply webpack.config.js:261]: Oh, I guess it's from an NPM package.

**Review by jryans — APPROVED** [review]
Changes here seem potentially okay, but we should if the `MediaRecorder` API can avoid the need for importing a recording lib altogether.

File: webpack.config.js (hunk 1 above)
Comment (turt2live) [reply webpack.config.js:263]: have added docs anyhow for the next person who reads this

**Pull request merged**